types/expo/index.d.ts

```diff
@@ -2561,13 +2699,14 @@ export class Svg extends Component<{ width: number, height: number, viewBox?: st
  * Take Snapshot
  */
 export function takeSnapshotAsync(
-    view?: (number | React.ReactElement<any>),
+    node: number | React.ReactElement<any> | React.RefObject<any>,
```

Comment:
Why was this changed to `node`? [In the documentation it's still `view`](https://docs.expo.io/versions/latest/sdk/take-snapshot-async/#takesnapshotasyncview-options). Are the docs outdated?

Reply:
@janaagaard75 I have copied most of the types here from their codebase. Some of the docs are outdated.

Reply:
Thanks for the link.